### ISSUE TITLE
stopEditingWhenGridLosesFocus=true popup editor

### ISSUE THREAD

**Issue by yang8807** [ag-grid/ag-grid#4634]
I configured aggrid to stopEditingWhenGridLosesFocus=true, and used a custom editor. My custom editor would pop up a Popover. When I clicked on an element in Popover, the aggrid was out of focus and ended editing. At this time, I don’t want to Let aggrid end editing;

![image](https://user-images.githubusercontent.com/10320021/130419103-994a9fa5-1509-4638-aba6-6d58214a2ac0.png)

version:26.0.0

chorme:  92.0.4515.159 for Mac

**Comment by makinggoodsoftware** [comment]
Hi,

 

This ticket has been flagged as managed-by-community/waiting-for-repro for a while now and there has not been any activity, to help us tidy up we are going to close it.

 

Thanks

**Comment by playinglife** [comment]
> I configured aggrid to stopEditingWhenGridLosesFocus=true, and used a custom editor. My custom editor would pop up a Popover. When I clicked on an element in Popover, the aggrid was out of focus and ended editing. At this time, I don’t want to Let aggrid end editing;
> 
> version:26.0.0
> 
> chorme: 92.0.4515.159 for Mac

I am also looking for a solution for this issue. If anyone has one please share.
Ideally I wanted to capture the focusout or blur event on the grid and prevent the default actions.
The steps would be these:

I start editing a cell with a custom cell editor.
In the custom cell editor component in the beforeMount life cycle, I wanted to disable the focus out event on the grid.
Then when editing is done I wanted to re-enable the focusout normal actions.

If anyone from staff could help me with an info on how this could be achieved it would be awesome.
Basically I need to know which focusout/blur event handler, handles the stopEditing so I can somehow prevent it.

Thank you

**Comment by metaljelly** [comment]
> 

https://www.ag-grid.com/javascript-data-grid/component-filter/#custom-filters-containing-a-popup-element

I solved it by adding "ag-custom-component-popup" class to the document.

Good luck

**Comment by MuXiTong** [comment]
> > 
> 
> https://www.ag-grid.com/javascript-data-grid/component-filter/#custom-filters-containing-a-popup-element
> 
> 我通过在文档中添加“ag-custom-component-popup”类来解决它。
> 
> 祝你好运

Thank you for solving my problem
